README.md

```diff
@@ -219,6 +219,7 @@ The `HTMLProofer` constructor takes an optional hash of additional options:
 | `check_favicon` | Enables the favicon checker. | `false` |
 | `check_opengraph` | Enables the Open Graph checker. | `false` |
 | `check_html` | Enables HTML validation errors from Nokogiri | `false` |
+| `check_img_http` | Fails an image if it's marked as http | `false` |
```

Comment:
You'll need to wrap this in `http` to get the test to pass. ✌️  (The test ensures the command line and README have the same documentation.)